fixtures/initialise_tables.sql

```diff
@@ -0,0 +1,74 @@
+-- Delete tables
+DROP TABLE IF EXISTS tags_challenges CASCADE;
+DROP TABLE IF EXISTS challenges CASCADE;
+DROP TABLE IF EXISTS tags_organisations CASCADE;
+DROP TABLE IF EXISTS organisations CASCADE;
+DROP TABLE IF EXISTS people CASCADE;
+
+-- TAGS --
```

Comment:
It not run on registration. It is an external initialiser, similar to what we have on ce100, with the addition of tags